11ty/_includes/layouts/base.njk

```diff
@@ -35,6 +35,29 @@
     {% endblock pageStyles %}
   </head>
   <body>
+    <script>
+      /**
+      * @type String
+      */
+      let userThemeSetting = localStorage.getItem('sdUserTheme');
+
+      if (userThemeSetting) {
+        document.documentElement.setAttribute(
+          'data-user-theme',
```

Comment:
Not sure if this was an intentional decision, but consider using [the `dataset` API](https://developer.mozilla.org/docs/Web/API/HTMLElement/dataset) for `data-` attributes on this and any other instances if possible.

Reply:
This code just moved. I didn't change any implementation details beyond timing.

Reply:
I think this was me, so feel free to update this, @eps1lon, but it’s on me to do it, if you don’t want to. :)

Reply:
Gotcha, no worries, then. No need to break things if they're working fine!